core/lib/Cardano/Address/Style/Byron.hs

```diff
@@ -205,18 +212,25 @@ type family DerivationPath (depth :: Depth) :: Type where
 -- > :set -XOverloadedStrings
 -- > :set -XTypeApplications
 -- > :set -XDataKinds
+-- > :set -XFlexibleContexts
 -- > import Cardano.Mnemonic ( mkSomeMnemonic )
+-- > import Cardano.Address ( base58 )
+-- > import Cardano.Address.Derivation ( toXPub )
+-- > import qualified Cardano.Address.Style.Byron as Byron
 -- >
 -- > let (Right mw) = mkSomeMnemonic @'[12] ["moon","fox","ostrich","quick","cactus","raven","wasp","intact","first","ring","crumble","error"]
--- > let rootK = genMasterKeyFromMnemonic mw :: Byron 'RootK XPrv
+-- > let rootK = Byron.genMasterKeyFromMnemonic mw :: Byron 'RootK XPrv
 --
 -- === Deriving child keys
 --
--- > let Just accIx = fromWord32 0x80000000
--- > let acctK = deriveAccountPrivateKey rootK accIx
+-- > let Just accIx = indexFromWord32 0x80000000
+-- > let acctK = Byron.deriveAccountPrivateKey rootK accIx
 -- >
--- > let Just addIx = fromWord32 0x80000014
--- > let addrK = deriveAddressPrivateKey acctK addIx
+-- > let Just addIx = indexFromWord32 0x80000014
```

Comment:
I wonder if there is a reason to use exactly this number (or any hardened index will do) ?
it would be nice to clarify.  

```suggestion
-- > let firstHardenedIndex = 0x80000000
-- > let Just addIx = indexFromWord32 (firstHardenedIndex + 14)
```

Reply:
added requested information in examples with exact ranges possible for addIx and acctIx in both byron and icarus cases